pkg/grpc/grpc.go

```diff
@@ -124,6 +124,10 @@ func (g *Manager) GetGRPCConnection(address, id string, namespace string, skipTL
 		return nil, err
 	}
 
+	if g.connectionPool[address] != nil {
```

Comment:
alternatively, we can do this
```
if c, ok := g.connectionPool[address]; ok {
    c.Close()
}
```

Reply:
Ah, yes, that the more goish way to do this.